src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+Microsoft.CodeAnalysis.VisualBasic.Syntax.SyntaxNodeRemover.New() -> Void
```

Comment:
All the public API changes are due to a breaking change in Public API analyzer that now requires implicitly declared public constructors to have a public API entry - see https://github.com/dotnet/roslyn-analyzers/issues/806 for details.